arvo/jael/scry.md

```diff
@@ -0,0 +1,222 @@
++++
+title = "Scry Reference"
+weight = 3
+template = "doc.html"
++++
+
+Jael's scry endpoints never take a `care`. The particular endpoints are specified in the place of the desk in the path prefix, then additional arguments are specified in the path.
+
+# Contents
+
+- [step](#step) - Web login code revision number.
+- [code](#code) - Web login code.
+- [life](#life) - Key revision number (`life`).
+- [lyfe](#lyfe) - Unitised `life`.
+- [rift](#rift) - Continuity number (`rift`).
+- [ryft](#ryft) - Unitised `rift`.
+- [vein](#vein) - Private key.
+- [vile](#vile) - Jammed `seed` boot parameters.
+- [deed](#deed) - Public key and related details.
+- [earl](#earl) - Derive moon keys.
+- [sein](#sein) - Sponsor.
+- [saxo](#saxo) - Sponsorship chain.
+- [subscriptions](#subscriptions) - Public key trackers.
+- [sources](#sources) - PKI sources.
+- [turf](#turf) - Galaxy domain prefixes.
+
+# `%step`
+
+A `%step` scry gets the current web login code revision number. It takes your ship name as its `path` and the type returned is a `@ud`.
+
+## Example
+
+```
+> .^(@ud %j /=step=/(scot %p our))
+0
+```
+
+# `%code`
+
+A `%code` scry gets the current web login code. It takes your ship name as its `path` and the type returned is a `@p`.
+
+## Example
+
+```
+> .^(@p %j /=code=/(scot %p our))
+~lidlut-tabwed-pillex-ridrup
+```
+
+# `%life`
+
+A `%life` scry gets the current `life` (key revision number). It takes a ship as its `path` and the type returned is a `@ud`.
```

Comment:
```suggestion
A `%life` scry gets the current `life` (key revision number) of a ship. It takes a ship as its `path` and the type returned is a `@ud`.
```

Same for `lyfe` below.

Reply:
Also, to distinguish this from `lyfe`, you might want to mention that this fails to produce a result if we do not know it (ie crash) whereas `lyfe` will always return a `unit`, but it might be empty if we don't know the answer.

Reply:
The above two comments apply to `rift` and `ryft` as well.